styles/style.css

```diff
@@ -92,12 +100,46 @@ p {
   border-left-color: var(--orange-dark);
 }
 
-
 /* Buttons */
 
-  /* INSERT BUTTON STYLES HERE */
+.button1 {
+  height: 35px;
+  width: 150px;
+  margin-left: 2.1rem;
+  border-radius: 6px;
+  color: var(--white);
+  background-color: var(--orange-dark);
+}
+.button1:hover {
+  background-color: white;
+  color: black;
+}
 
+.button2 {
+  height: 35px;
```

Comment:
Could you re-use styles in your button classes? eg. you have given `.button1`, `button2` and `button3` each a height of `35px`. Can you work out how to avoid repeating the `height` property in each class? Have a look at https://www.educative.io/edpresso/how-to-use-multiple-classes-on-the-same-element-in-css